### ISSUE TITLE
What do you use for validation?

### ISSUE THREAD

**Issue by KaKi87** [sholladay/pogo#67]

**Comment by sholladay** [comment]
I use [joi](https://joi.dev).

```js
import joi from 'https://jspm.dev/joi';
```

At the moment, Pogo has no special handling of joi schemas or error objects, but that's something I intend to implement. PRs welcome if you want it implemented more quickly.

**Comment by KaKi87** [comment]
I'm using Joi from Node, but I didn't realized it was possible to use it out-of-the-box from Deno, so I went to the trouble of looking for an alternative on deno.land, without much success.

But, yes, I'm actually satisfied with using Joi on Deno.

> I intend to implement a system like hapi has, where you can provide a schema for the request and response as route options and Pogo will handle the validation automatically to simplify your code

Well, I created my own `createRoute` function at [`@cv.vg/api/lib/route.js`](https://git.kaki87.net/cv.vg/api/src/branch/dev/lib/route.js) which handles that, as well as CORS.
